userspace/libsinsp/sinsp_suppress.cpp

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2023 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+#include "sinsp_suppress.h"
+
+#include "sinsp_exception.h"
+#include "../../driver/ppm_events_public.h"
+#include "scap_const.h"
+#include "scap_assert.h"
+
+void libsinsp::sinsp_suppress::suppress_comm(const std::string &comm)
+{
+	m_suppressed_comms.emplace(comm);
+}
+
+void libsinsp::sinsp_suppress::suppress_tid(uint64_t tid)
+{
+	m_suppressed_tids.emplace(tid);
+}
+
+bool libsinsp::sinsp_suppress::check_suppressed_comm(uint64_t tid, const std::string &comm)
+{
+	if(m_suppressed_comms.find(comm) != m_suppressed_comms.end())
+	{
+		m_suppressed_tids.insert(tid);
+		m_num_suppressed_events++;
```

Comment:
Where is `m_num_suppressed_events` used?

Reply:
Right. Nowhere ;) Will fix.